spec/rails_helper.rb

```diff
@@ -280,6 +280,33 @@ def raise_server_error!
 
     Capybara::Session.class_eval { prepend IgnoreUnicornCapturedErrors }
 
+    module MatcherTimeoutExtension
+      class MatcherTimedOut < StandardError
+        def initialize(wait_time)
+          super "Matcher waited for the full wait duration (#{wait_time}s). " +
```

Comment:
I’m not sure if this is related to RSpec marchers only. I think we want to raise an error as long as Capybara’s finder waits the full timeout?

Reply:
I'm not super familiar with the terminology here. Are you suggesting we need some functional change? Or just that the message copy needs to be tweaked? What would you suggest?

Reply:
Oops sorry I missed your reply here. What I mean is that Capybara's timeout is hit as long as finder methods are used but the error message we're raising here stated that the `Matcher` waited for the full wait duration when it could just be any capybara finder method. Maybe just "Capybara waited for the full wait duration"?